sqlglot/expressions.py

```diff
@@ -4118,13 +4118,6 @@ class Bracket(Condition):
     # https://cloud.google.com/bigquery/docs/reference/standard-sql/operators#array_subscript_operator
     arg_types = {"this": True, "expressions": True, "offset": False, "safe": False}
 
-    @property
-    def output_name(self) -> str:
```

Comment:
why'd you get rid of this?

Reply:
it was originally [introduced](https://github.com/tobymao/sqlglot/pull/2186) so that we'd produce `field` as the alias for `SELECT table.column:field` when qualifying Snowflake queries. Thought I'd get rid of it since the representation changed but it's not necessary - can bring it back.

Reply:
let's bring it back

Reply:
we still have x['y'] right

Reply:
Ah, makes sense. Brought it back.